amarachukwu-agbo/population-manager
# #165484472 Create endpoint to add locations

#### What does this PR do?
Creates endpoint to add locations

#### Description of Task to be completed?
- Create controller for adding location
- Create route for adding location
- Create middlewares for validation
- Create model and migration for location
- Create jest config and babel config file
- Write tests

#### How should this be manually tested?
- Clone the repo and install application dependencies.
- Open Postman and make a `POST` request to `http://localhost:6000/api/v1/locations` with the body
`{
	"name": "Another location",
	"maleOccupantCount": 20,
	"femaleOccupantCount": 40,
	"parentLocationId": 6
}`
- The location should be created.
- Run command  `yarn test`. 
- You should see tests running.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#165484472](https://www.pivotaltracker.com/story/show/165484472)

#### Screenshots (if appropriate)
N/A

#### Questions: 
N/A
